sympy/integrals/integrals.py

```diff
@@ -153,6 +153,60 @@ def _eval_is_zero(self):
         if self.function.is_zero is False and got_none is False:
             return False
 
+    def is_divergent(self):
```

Comment:
What about naming it `is_convergent`? `Sum` and `Product` both have `is_convergent` but not `is_divergent`. It might be better to keep things consistent.

Reply:
Yeah i was doing that earlier but the problem is if the tests pass then we can say that the integral is divergent but if the tests fail we can't say that the integral is convergent so wouldn't it be wrong to have a function called `is_convergent` which never returns True

Reply:
We can use three valued boolean such that `None` can be used for nonconvergent but also not sure about divergent

Reply:
The divergence test is only one of many. `is_convergent` or `is_divergent` should implement as many as possible (as `Sum` does). Returning false and none is still as useful as returning true and none. False means that we know it is not convergent.

Reply:
ok got it, so after changing function to `is_convergent` with the current test the only possible values will be `False` when tests pass and we find that it is divergent and `None` if the tests fail and we are not sure about divergent, convergent.

Reply:
> the only possible values will be False when tests pass and we find that it is divergent

If change the function to be centered around the integral test, then we can use `Sum.is_convergent` which returns true false and none.

> if the tests fail and we are not sure about divergent, convergent

That would have been the case if we keep it `is_divergent` as well. All we should do is simply negate the output of `is_divergent` and change its name.